core/src/main/java/io/grpc/Channel.java

```diff
@@ -49,14 +49,30 @@
 
   /**
    * Create a {@link ClientCall} to the remote operation specified by the given
+   * {@link MethodDescriptor}, and with the default call options.
+   *
+   * @param methodDescriptor describes the name and parameter types of the operation to call.
+   * @return a {@link ClientCall} bound to the specified method.
+   * @deprecated use {@link newCall(MethodDescriptor, CallOptions)}
+   *
+   */
+  @Deprecated
+  public final <RequestT, ResponseT> ClientCall<RequestT, ResponseT> newCall(
```

Comment:
@ejona86 I have added the old newCall() back to allow an easier transition for its callers.